src/pages/EditCategory/EditCategory.jsx

```diff
@@ -44,34 +38,38 @@ export default function EditCategory({ id, title }) {
     }
 
     return (
-        <>
-            <h1>{title}</h1>
-            <form
-                action="#"
-                className="newCategoryForm twoColumns"
-                onSubmit={handleSubmit}
-            >
-                <label id="description">
-                    Descrição<br />
-                    <input
-                        onChange={handleTextInputChange}
-                        type="text"
-                        name="name"
-                        value={category?.name}
-                    />
-                </label>
-                <label id="enabled">
-                    Habilitado?<br />
-                    <input
-                        onChange={handleCheckboxChange}
-                        checked={category?.state === "A"}
-                        name="state"
-                        type="checkbox"
-                    />
-                </label>
-                <button className="buttonCancelar">Cancelar</button>
-                <button type="submit" className="buttonSalvar">Salvar</button>
-            </form>
-        </>
+        <form 
+            className={styles.form}
+            onSubmit={handleSubmit}
+        >
+            <div className={styles.formContainer}>
+                <TextInput                  
+                    labelText="Descrição" 
+                    onChange={handleTextInputChange}
+                />
+                <fieldset>
+                    <legend>Tipo</legend>
+                    <div className={styles.formRadioInputContainer}>
+                        <RadioInput inputName="expense-type" labelText="Despesa" checked />
+                    </div>
+                    <div className={styles.formRadioInputContainer}>
+                        <RadioInput inputName="expense-type" labelText="Receita" />
+                    </div>
+                </fieldset>
+                <fieldset>
+                    <legend>Status</legend>
+                    <div className={styles.formRadioInputContainer}>
+                        <RadioInput inputName="expense-status" labelText="Habilitado" checked />
+                    </div>
+                    <div className={styles.formRadioInputContainer}>
+                        <RadioInput inputName="expense-status" labelText="Desabilitado" />
+                    </div>
+                </fieldset>
+            </div>
+            <div className={styles.formButtonsContainer}>
+                <Button type="submit" primary>Salvar</Button>
+                <Button type="submit" secondary>Voltar</Button>
```

Comment:
```suggestion
                <Button type="button" secondary>Voltar</Button>
```

Sugiro remover o `type="submit"` desse botão **Voltar**, para que ele não submita o formulário quando for ativado.

Reply:
Bem observado, acabou passando batido